GAE/src/org/waterforpeople/mapping/app/web/SurveyedLocaleServlet.java

```diff
@@ -182,19 +179,16 @@ protected SurveyedLocaleResponse convertToResponse(List<SurveyedLocale> slList,
                                 // if the question has the allowOtherFlag set,
                                 // use OTHER as the device question type
                                 if (q.getAllowOtherFlag()) {
-                                    deviceQuestionType = "OTHER";
+                                    type = "OTHER";
                                 }
-                                questionTypeMap.put(sv.getSurveyQuestionId(),
-                                        deviceQuestionType);
+                                questionTypeMap.put(sv.getSurveyQuestionId(), type);
                             }
                         }
                     }
-                    if (!sv.getQuestionType().equals("IMAGE")) {
-                        // add question type
-                        siDto.addProperty(sv.getSurveyQuestionId(),
-                                sv.getStringValue() != null ? sv.getStringValue() : "",
-                                deviceQuestionType);
-                    }
```

Comment:
Image *responses* have always been synced. Images themselves were synced in more recent versions, but this feature is completely unrelated.